recipes/sdl2_mixer/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "SDL2_mixer" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "5a24f62a610249d744cbd8d28ee399d8905db7222bf3bdbc8a8b4a76e597695f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.libsdl.org/projects/SDL_mixer/release/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    - cmake  # [win]
+    - pkgconfig  # [unix]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - toolchain
+    - sdl2 2.0.*
+    - libogg 1.3.*
+    - libvorbis 1.3.*
+    - libflac 1.3.*
+    - libmad 0.15.*
```

Comment:
Yes, I agree that it is better to link dynamically, and yes, on linux it always does so. However, since the build/configure script looks for these libraries, I thought it was smarter to include them at build time, even though I don't know if it has any benefits/consequences for the resulting build.

On Windows on the other hand, it can be more difficult because sdl2_mixer doesn't always look for the libraries at the correct places. I was advised to compile sdl2_mixer against the static libraries, as I do here. And, as far as I know, sdl2_mixer first tries to look for and use the dynamic libraries and uses these statically linked libraries as a callback if it can't find them.